reports-scheduler/src/main/config/reports-scheduler.yml

```diff
@@ -27,24 +27,7 @@
 ##
 
 # configuration file for the reports scheduler plugin
-opendistro.reports:
+opensearch.reports:
```

Comment:
would this break upgrades from opendistro?

Reply:
we are completely removing support for filtering based on this setting because we need to support compatibility with notification plugin. Remaining settings are relevant for admin/maintainers only